api/BluetoothRemoteGATTService.json

```diff
@@ -467,54 +371,6 @@
           }
         }
       },
-      "isPrimary": {
```

Comment:
It looks like this is actually supported as per https://mdn-bcd-collector.appspot.com/tests/api/BluetoothRemoteGATTService/isPrimary.  Can we revert this change?